helpers/apolloClient.ts

```diff
@@ -12,6 +12,25 @@ export const APOLLO_STATE_PROP_NAME = '__APOLLO_STATE__'
 let apolloClient: ApolloClient<NormalizedCacheObject>
 const cache = new InMemoryCache()
 
+import { persistCache } from 'apollo3-cache-persist'
+//restoring cache doesn't work with loading spinner, so I need to enable cache page by page
+const whiteList = ['/curriculum']
```

Comment:
Loading spinner doesn't work with persistent cache, hence white list. Eventually I'm going to remove loading spinner everywhere.